qmarkdowntextedit.cpp

```diff
@@ -22,6 +22,9 @@
 #include <QKeyEvent>
 #include <QLayout>
 #include <QPainter>
+#if QT_VERSION >= QT_VERSION_CHECK(5, 15, 0)
```

Comment:
I dont think the #ifdefs are necessary.

Reply:
Possibly...

Reply:
After upgrading Qt I skipped many versions of QOwnNotes from AUR.
So I decided made the patch.

Reply:
Right, but this is not a new header, so the `#if` is not needed.

Reply:
@bjorn #113 ;)